core/state_processor.go

```diff
@@ -57,9 +57,10 @@ var (
 )
 
 const (
-	receiptsCacheLimit = 32
-	txLookupCacheLimit = 1024
-	TriesInMemory      = 128
+	receiptsCacheLimit      = 32
+	txLookupCacheLimit      = 1024
+	TriesInMemory           = 128
+	inboundEtxExpirationAge = 8640 // With 10s blocks, ETX expire after ~24hrs
```

Comment:
saw this in some other commit I believe. If i can find will drop the reference.

Reply:
@gameofpointers did you find it?

Reply:
its defined in the EtxSet (core/types/etx_set.go). I guess you cannot use that here!.

Reply:
Its no longer used, actually. Removed it: a05576e